src/agent/testresultreader.ts

```diff
@@ -240,6 +240,9 @@ export class ResultReader {
                 outcome = "Failed";
                 errorMessage = testCaseNode.error.text();
             }
+            else if (testCaseNode.skipped) {
+                outcome = 'NotExecuted';
```

Comment:
Can we use "" to keep style consistent? Will be good to set errorMessage = testCaseNode.skipped.text() for cases where @Ignore() tag has some message.